salt/cloud/clouds/aliyun.py

```diff
@@ -339,7 +339,7 @@ def list_nodes_full(call=None):
         provider = comps[0]
 
     __opts__['update_cachedir'] = True
-    salt.utils.cloud.cache_node_list(ret, provider, __opts__)
+    __utils__['cloud.cache_node_list'](ret, provider, __opts__)
```

Comment:
I left that in just in case someone else was using it for one of their functions in reference to something.

And then we can remove it in Carbon, instead of making a change to the function that we don't need to make back in an older release.  But I can remove the __opts__ if that is what should be done.